ssl/ssl_cert.c

```diff
@@ -197,10 +218,10 @@ CERT *ssl_cert_dup(CERT *cert)
 
 void ssl_cert_clear_certs(CERT *c)
 {
-    int i;
+    size_t i;
     if (c == NULL)
```

Comment:
Blank line between declarations and code.

Reply:
so changed in several locations in https://github.com/openssl/openssl/pull/19312/commits/e5582173f0db49a28edf0907db053ea66e10d3dc